examples/snapshot_backup/list_project_snapshot_backups/README.md

```diff
@@ -0,0 +1,506 @@
+# List Existing Project Level Cloud Snapshot Backups
+
+This example shows how to list cloud snapshot backups that already exist in Capella for a given cluster. It uses the organization ID and project ID to do so. 
+
+To run, configure your Couchbase Capella provider as described in README in the root of this project.
+
+## List
+### View the plan for the resources that Terraform will create
+
+Command: `terraform plan`
+
+Sample Output:
+
+```
+
+ terraform plan
+╷
+│ Warning: Provider development overrides are in effect
+│ 
+│ The following provider development overrides are set in the CLI configuration:
+│  - couchbasecloud/couchbase-capella in /Users/laura.silaja/code/Lagher0/terraform-provider-couchbase-capella/bin
+│ 
+│ The behavior may therefore not match any released version of the provider and applying changes may cause the state to become incompatible with published releases.
+╵
+data.couchbase-capella_cloud_project_snapshot_backups.existing_cloud_project_snapshot_backups: Reading...
+data.couchbase-capella_cloud_project_snapshot_backups.existing_cloud_project_snapshot_backups: Read complete after 0s
+
+Changes to Outputs:
+  + project_backups_list = {
+      + cursor          = {
+          + hrefs = {
+              + first    = "http://localhost:8084/v4/organizations/ffffffff-aaaa-1414-eeee-000000000000/projects/ffffffff-AAAA-1414-eeee-000000000000/cloudsnapshotbackups?page=1&perPage=2"
+              + last     = "http://localhost:8084/v4/organizations/ffffffff-aaaa-1414-eeee-000000000000/projects/ffffffff-AAAA-1414-eeee-000000000000/cloudsnapshotbackups?page=1&perPage=2"
+              + next     = "http://localhost:8084/v4/organizations/ffffffff-aaaa-1414-eeee-000000000000/projects/ffffffff-AAAA-1414-eeee-000000000000/cloudsnapshotbackups?page=0&perPage=2"
```

Comment:
Can you replace all instances of `localhost` with the URL for prod capella `https://cloudapi.cloud.couchbase.com`